packages/web_ui/src/components/InstanceStatusTag.jsx

```diff
@@ -0,0 +1,21 @@
+import { Tag } from "antd";
+import React from "react";
+
+const statusColors = {
+	unknown: "#d46b08",
+	unassigned: "#d4380d",
+	stopped: "#cf1322",
+	starting: "#7cb305",
+	running: "#389e0d",
+	stopping: "#d48806",
+	creating_save: "#096dd9",
+	exporting_data: "#08979c",
```

Comment:
IMO creating a save and exporting data are both very linked operations ( Working with data, writing to disk, etc ); perhaps they should be closer in color? #096dd9 ( dark blue-ish ) looks fine to me for creating_save, but I would suggest #69c0ff ( Lighter blue ) for exporting_data, keeping them both shades of blue as opposed to the contrasting #08979c ( teal-ish ) which feels like it's something completely different.

Reply:
In this case they are entirely different operations though. Exporting_data scrapes lua prototype data and images and is more of a clusterwide task, while create_save is more of an instance setup/startup task. I think this is fine.